src/database-layer/db.ts

```diff
@@ -106,8 +106,8 @@ const atomicExecuteSql: Database['executeSql'] = function(sql, bindings) {
 	return this.transaction(tx => tx.executeSql(sql, bindings));
 };
 
-const tryFn = (fn: () => any) => {
-	Promise.try(fn);
+const asyncTryFn = (fn: () => any) => {
+	Promise.resolve().then(fn);
```

Comment:
What is the actual difference with the previous variant?

Reply:
This will schedule `fn` to be called asynchronously rather than synchronously